PyU4V/replication.py

```diff
@@ -863,38 +867,38 @@ def modify_storage_group_srdf(
         'ReplicationFunctions', 'suspend_storage_group_srdf', 91, 93)
     def suspend_storagegroup_srdf(self, storagegroup_id, rdfg_no,
                                   suspend_options=None, _async=False):
-        """Suspend IO on the links for the given storage group.
+        '''Suspend IO on the links for the given storage group.
 
         DEPRECATION NOTICE: ReplicationFunctions.suspend_storagegroup_srdf()
         will be refactored in PyU4V version 9.3 in favour of
         ReplicationFunctions.suspend_storage_group_srdf(). For further
         information please consult PyU4V 9.1 release notes.
 
-        Optional boolean parameters to set are "bypass", "metroBias", "star",
-        "immediate", "hop2", "consExempt", "force", "symForce".
+        Optional boolean parameters to set are 'bypass', 'metroBias', 'star',
+        'immediate', 'hop2', 'consExempt', 'force', 'symForce'.
```

Comment:
Just to avoid confusion.  Having single quotes here is fine. It is also in the doc strings so less concerned with this anyway

Reply:
I've changed back to original, this change was made in error